tools/wpt/testfiles.py

```diff
@@ -299,7 +299,7 @@ def affected_by_wdspec(test):
 
     def affected_by_interfaces(file_contents):
         # type: (Union[bytes, Text]) -> bool
-        if len(interfaces_changed_names) > 0:
+        if interfaces_changed_names:
```

Comment:
This seems wrong; in Python 3:
```
In [4]: bool(map(lambda x: x, []))
Out[4]: True
```

i.e. this will evaluate as True even if the `interfaces_changed_names` map is empty, because the iterator will always evaluate as True. I think you could do `if interfaces_changed` instead?

Reply:
Good point. I'll change it